include/sdf/Error.hh

```diff
@@ -135,6 +135,9 @@ namespace sdf
     /// \brief The pose relative-to graph has an internal error.
     POSE_RELATIVE_TO_GRAPH_ERROR,
 
+    /// \brief The rotation snap config provided is not valid.
+    ROTATION_SNAP_CONFIG_ERROR,
```

Comment:
inserting an enum value breaks API; I'll split this out into a separate PR before feature freeze

Reply:
#1099